### PR TITLE
[RU] Fixed new line and small adjustment

### PR DESCRIPTION
**Pull request by REALMWTH**
# SCP:SL Translation PR Template

## Language Name
Russian

## State Type of change (New language, changed existing Etc.)
Changed existing language. Fixed new line and adjusted a little text. Also reflected the Badges.txt update.

# Checklist:
- [x] I have tested the translation changes ingame
- [x] I have made sure to not commit changes to multiple translations at the same time, or non-translation repo files
- [x] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  (Exception to this is when edits has __only__ been made in-browser. If this is the case, also check this box.)
- [x] I have made sure that I'm **not** submitting a duplicate translation 
- [x] I have made sure to **not** included any advertisements/pointers to non-game stuff other than the translation author in the credits file
- [x] I have made sure to label my translation folder by [Microsoft Language Tag Standards](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c)
   - Only in cases of **new** translations (Can be tested [Here](https://rextester.com/WDXPS97501)). Also check this box if the change is **not** a new translation

### CODE REVIEW

File: Translations/ru/MenuWarnings.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ВНИМАНИЕ
 ОК
-Было обнаружено <b>неподдерживаемое</b> устройство <b>Intel graphics</b>./n<b>Рекомендуем</B> использовать <b>дискретную</b> видеокарту./nНа вашем устройстве игра может <b>часто вылетать</b>.
+Было обнаружено <b>неподдерживаемая</b> встроеная видеокарта от <b>Intel Graphics</b>.\n<b>Рекомендуем</B> использовать <b>дискретную</b> видеокарту.\nНа вашем устройстве игра может <b>часто вылетать</b>.
```
Comment (MichalPetryka) [inline Translations/ru/MenuWarnings.txt:3]:
```
<b>Рекомендуем</B>
```
Don't think the 2nd b should be uppercase

Comment (GuyFawkes76) [reply Translations/ru/MenuWarnings.txt:3]: "Был**о** обнаружен**о**" - "неподдерживаем**ая** встроен**ая**".
Change either "встроеная видеокарта" to smth else or, a better way, change "Был**о** обнаружен**о**" to "Был**а** обнаружен**а**"

Comment (REALMWTH) [reply Translations/ru/MenuWarnings.txt]: fixed both of issues

**Comment by REALMWTH** [comment]
Reflected this pull request:
https://github.com/northwood-studios/SCPSL-Translations/pull/592/commits/23a55b904a534b7d349228aedd17efa4db4bc444

**Review by GuyFawkes76 — APPROVED** [review]

**Pull request merged**